vlm/src/main/scala/geotrellis/contrib/vlm/gdal/GDAL.scala

```diff
@@ -88,8 +90,11 @@ object GDAL {
       case _ => throw new Exception(s"Could not find equivalent GDALResampleMethod for: $method")
     }
 
-  def open(path: String): Dataset = {
-    val ds = gdal.Open(path, gdalconstConstants.GA_ReadOnly)
+  def open(path: String): Dataset =
+    open(new URI(path))
+
+  def open(uri: URI): Dataset = {
```

Comment:
@pomadchin do you remember exactly why we had to avoid the ‘URI’ container? https://github.com/raster-foundry/raster-foundry/blob/develop/app-backend/common/src/main/scala/utils/RangeReaderUtils.scala Something about a wacky s3 uri violated the character constraints and I am not sure if it would be an issue here, generating vsipath from such uri

Reply:
@echeipesh i don't remember actually, but yeah there was smth with it, I'll double check with James about RF case

Reply:
@echeipesh 
```
James:
one possible reason -- the `%7C` / `|` thing because we're using auth0 users for a component of the path
```

^ that's definitely smth i remember was a problem in RF, and it was related to how `URI` handles special characters.